packages/mobile/src/invite/saga.ts

```diff
@@ -1,5 +1,5 @@
 import { CeloTransactionObject } from '@celo/contractkit'
-import { RpcWallet } from '@celo/contractkit/lib/wallets/rpc-wallet'
+import { WritableWallet } from '@celo/contractkit/lib/wallets/wallet'
```

Comment:
Would there be a better name for this?
From the outside I don't really see what it does/can do.

Reply:
Yeah I was hoping somebody comes up with a better name for this during the review :D so the interfaces look like this:
```typescript
export interface Wallet {
    getAccounts: () => Address[];
    hasAccount: (address?: Address) => boolean;
    signTransaction: (txParams: Tx) => Promise<EncodedTransaction>;
    signTypedData: (address: Address, typedData: EIP712TypedData) => Promise<string>;
    signPersonalMessage: (address: Address, data: string) => Promise<string>;
    decrypt: (address: Address, ciphertext: Buffer) => Promise<Buffer>;
}
export interface WritableWallet extends Wallet {
    isAccountUnlocked: (address: string) => boolean;
    unlockAccount: (address: string, passphrase: string, duration: number) => Promise<boolean>;
    addAccount: (privateKey: string, passphrase: string) => Promise<string>;
}
```

The writable wallet supports adding accounts to it and unlocking accounts. Essentially it supports modifying it's state (writing) while the base Wallet is read-only. Do you have any suggestions name wise?

Reply:
Naming is hard 😄 
I don't have a perfect name for now, one that comes to mind is `MutableWallet`, but still very similar.

Reply:
Something else I was considering was maybe renaming the base and have:
- `Wallet extends ReadOnlyWallet` or
- `Wallet extends StatelessWallet` or
- `Wallet extends ImmutableWallet`

What do you think?

Reply:
Good idea! To me looks like `ReadOnlyWallet` could be the better option.